tests/test_core.py

```diff
@@ -19,8 +21,16 @@ class UnittestEnv(core.Env):
     observation_space = spaces.Box(low=0, high=255, shape=(64, 64, 3), dtype=np.uint8)
     action_space = spaces.Discrete(3)
 
-    def reset(self, *, seed: Optional[int] = None, options: Optional[dict] = None):
+    def reset(
+        self,
+        *,
+        seed: Optional[int] = None,
+        return_info: bool = False,
+        options: Optional[dict] = None
+    ):
```

Comment:
Return type?

Reply:
What do you mean?

Reply:
Missing type hint for the return values? (`Union[ObsType, Tuple[ObsType, dict]]`)

Reply:
That's automatically inherited from `gym.Env`, and individual envs don't currently have return annotations, so I don't think it's necessary